.github/workflows/test.yml

```diff
@@ -5,7 +5,19 @@ name: Tests
 
 on:
   pull_request:
+    paths-ignore:
+      - "docs/**"
+      - ".github/workflows/*.yaml"
+      - "!.github/workflows/publish.yaml"
```

Comment:
Any reason not to ignore the publish workflow here?

```suggestion
```

Reply:
Nice catch! It was copy pasted from a workflow named publish, but I had forgo to update it to de-ignore the `test.yaml` workflow. The idea was that "ignore all workflows (line before), but not this workflow"